functional_tests/scylla_operator/libs/helpers.py

```diff
@@ -230,3 +238,22 @@ def verify_resharding_on_k8s(db_cluster: ScyllaPodCluster, cpus: Union[str, int,
         assert not liveness_probe_failures_list, (
             f"liveness probe has failures: {liveness_probe_failures_list}")
     log.info("Resharding has successfully ended on whole Scylla cluster.")
+
+
+# TODO: add support for the single column output which doesn't have the "|" symbol
+def parse_cqlsh_select_output(cqlsh_output: str):
```

Comment:
this seems like a general purpose helper for using cqlsh, and not something specific to the functional tests, we should consider move it to it's own module under sdcm.utils

Reply:
also to simplify, could use `select json` instead parsing lines like this.

Reply:
@soyacz Indeed! I will use the `select json`. Thanks